docs/_drafts/2017-05-01-ReleaseMvvmCross5.md

```diff
@@ -11,24 +11,36 @@ For the last 6 months we have been working on this release and we're really exci
 
 Let's have a look at a couple of things:
 
-## Highlights
+# Highlights
```

Comment:
I'm not a fan of having a empty heading with no text underneath them. For me we can just remove the "Highlights" heading